src/Components/GetCountries.js

```diff
@@ -1,38 +1,58 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import axios from "axios";
 import GetCountryDataById from "./GetCountryDataById";
 
-export default class GetCountries extends React.Component {
-  state = {
-    Countries: [],
-    SelectedCountry: [],
-  };
-
-  componentDidMount() {
-    axios.get(`https://api.covid19api.com/countries`).then(res => {
-      this.setState({Countries: res.data});
-    })
-  }
+function GetCountries(props) {
+  const [selectedCountry, setSelectedCountry] = useState(null);
+  const [error, setError] = useState(null);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [items, setItems] = useState([]);
 
-  handleChange = (event) => {
-    const CurrentCountry = event.target.value;
-    this.setState({ IsLoading: true }, () => {
-      axios.get(`https://api.covid19api.com/total/dayone/country/${CurrentCountry}/status/confirmed`)
-        .then(result => this.setState({
-          IsLoading: false,
-          SelectedCountry: [...result.data],
-        }));
-    });
+  const fetchCountries = () => {
+    axios.get("https://api.covid19api.com/countries")
+      .then(
+        (res) => {
+          setIsLoaded(true);
+          setItems(res.data);
+        },
+        (error) => {
+          setIsLoaded(true);
+          setError(error);
+        }
+      )
   };
 
-  render() {
+  useEffect(() => {
+    fetchCountries();
+  }, []);
+
+  if (error) {
+    return <div>Error: {error.message}</div>;
+  } else if (!isLoaded) {
+    return <div>Loading...</div>;
+  } else {
+    let selectedLabel, selectedComponent;
+    if (selectedCountry) {
+      selectedLabel = <div className={'current-country'}>Current Country:&nbsp;<strong> {selectedCountry}</strong></div>;
```

Comment:
```suggestion
      selectedLabel = <div className="current-country">Current Country:&nbsp;<strong> {selectedCountry}</strong></div>;
```

You don't need the curly brackets here :)